PayloadPkg/OsLoader/BootOption.c

```diff
@@ -52,9 +52,30 @@ PrintBootOptions (
                  BootOption->Image[0].LbaImage.LbaAddr \
                  ));
     }
-  }
+    //Print Pre-OS image filename

```

Comment:
can we use a loop to avoid code duplication for the print for PreOS and Extra image?